doc/ref_arch/openstack/chapters/chapter02.rst

```diff
@@ -3666,18 +3660,18 @@ Compliance with Standards (source `RM 7.9.10 <../../../ref_model/chapters/chapte
 |           |                   | /12/20/version2-com |               |
 |           |                   | munity-release/>`__ |               |
 +-----------+-------------------+---------------------+---------------+
-| se        | Standards         | The Cloud Operator, |               |
+| se\       | Standards         | The Cloud Operator, |               |
 | c.std.006 |                   | Platform and        |               |
 |           |                   | Workloads           |               |
 |           |                   | **should** utilise  |               |
 |           |                   | the `OWASP Web      |               |
 |           |                   | Security Testing    |               |
-|           |                   | Guid                |               |
-|           |                   | e <https://github.c |               |
+|           |                   | Guide               |               |
+|           |                   | <https://github.c   |               |
```

Comment:
```suggestion
|           |                   | <https://github.c\   |               |
```

Reply:
@pgoyal01 suggestion for the word "Guide" integrated.
No backslash needed into URL.